libr/anal/anal.c

```diff
@@ -123,6 +123,11 @@ R_API void r_anal_plugin_free (RAnalPlugin *p) {
 	if (p && p->fini) {
 		p->fini (p);
 	}
+	/*
+	 * This could lead to a double free in the case a plugin
+	 * implements a fini function with a free to the plugin itself
+	 */
+	free (p);
```

Comment:
What do you think about removing the free of the plugin itself in all current plugins?